server/handlers/default.go

```diff
@@ -35,6 +37,76 @@ func MainHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) *e
 	return nil
 }
 
+// AtomicUpdateHandler will accept multiple TUF files and ensure that the storage
+// backend is atomically updated with all the new records.
+func AtomicUpdateHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) *errors.HTTPError {
+	defer r.Body.Close()
+	s := ctx.Value("metaStore")
+	if s == nil {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
+			Err:        fmt.Errorf("Version store is nil"),
+		}
+	}
+	store, ok := s.(storage.MetaStore)
```

Comment:
Because we're going to call methods on it that are defined on the MetaStore interface. I could move this down to just above line 99, but it's much cleaner to keep this generic setup here at the top.